.github/workflows/build.yml

```diff
@@ -84,6 +85,10 @@ jobs:
             RUNPOSTGRES="--runpostgres"
           fi
           pytest src/tests --runui $RUNPOSTGRES
+      - name: Run pytest on Windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          pytest src/tests --runui --runpostgres
```

Comment:
Does it have to be a separate step? Looks the same as the previous one

Reply:
In POSIX step POSIX shell syntax is used (`if [ ... ]; then`), on Windows runner PowerShell is used as a shell: https://github.com/un-def/dstack/actions/runs/10793434509/job/29935425474#step:6:19